source/stylesheets/modules/_search_box.scss

```diff
@@ -0,0 +1,31 @@
+@import "bourbon";
+@import "mixins/accessibility";
+@import "settings/colors";
+@import "settings/grid_layout";
+
+.search-box {
+
+  > .headline {
+    @include screenreaders-only;
+  }
+
+  input {
+    display: block;
+    width: 100%;
```

Comment:
What I meant was: `width: 100%` should be the default for `display: block` elements, so I think this is redundant.